src/Swarm/TUI/View.hs

```diff
@@ -544,12 +544,12 @@ drawKeyMenu s =
     [(NoHighlight, "F1", "help"), (NoHighlight, "F2", "robots")]
       <> notificationKey availableRecipes "F3" "Recipes"
       <> notificationKey availableCommands "F4" "Commands"
-      <> [(NoHighlight, "Tab", "cycle")]
       <> [(NoHighlight, "^v", "creative") | cheat]
       <> [(NoHighlight, "^g", "goal") | goal]
 
   keyCmdsFor (Just REPLPanel) =
     [ ("↓↑", "history")
+    , ("Tab" "Tab Completion")
```

Comment:
I think we probably don't need tab completion listed in the help bar for the REPL; tab completion is pretty standard and many users will probably just figure it out by trying it.  Also, this seems to be missing comma which is causing compilation to fail.
```suggestion
```